packages/vckit-oa-renderers/package.json

```diff
@@ -49,6 +49,7 @@
     "@types/jest": "^24.0.23",
     "@types/qrcode.react": "^1.0.1",
     "@types/react": "^16.9.11",
+    "@types/react-dom": "^16.9.0",
```

Comment:
why additional dependencies added when just importing the base template repo into pnpm workspace?